src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1052 @@ metadata_pkg.(
     is_wrapper = false
 )
 
+"""
+$(MMI.doc_header(LinearRegressor))
+
+`LinearRegressor` assumes the target is a continuous variable
+whose conditional distribution is normal with constant variance, and whose
+expected value is a linear combination of the features. Linear coefficients
+are calculated using least squares.
+Options exist to specify a bias term.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `Continuous`; check the scitype with `scitype(y)`
+
+# Hyper-parameters
+
+- `bias=true`: Include the bias term if true, otherwise fit without bias term.
+
+# Operations
+
+- `predict(mach, Xnew)`: Return predictions of the target given new
+  features `Xnew` having the same Scitype as `X` above.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `coefficients`: The linear coefficients determined by the model.
+- `intercept`: The intercept determined by the model.
+
+# Examples
+
+```
+using MLJ
+
+LinearRegressor = @load LinearRegressor pkg=MultivariateStats
+linear_regressor = LinearRegressor()
+
+X, y = make_regression(100, 2) # synthetic data
+mach = machine(linear_regressor, X, y) |> fit!
+
+Xnew, _ = make_regression(3, 2)
+yhat = predict(mach, Xnew) # new predictions
+```
+
+See also
+[`MultitargetLinearRegressor`](@ref), [`RidgeRegressor`](@ref), [`MultitargetRidgeRegressor`](@ref)
+"""
+LinearRegressor
+
+"""
+$(MMI.doc_header(MultitargetLinearRegressor))
+
+`MultitargetLinearRegressor` assumes the target is a continuous variable
+whose conditional distribution is normal with constant variance, and whose
+expected value is a linear combination of the features. Linear coefficients
+are calculated using least squares. In this case, the output represents a
+response vector.
+Options exist to specify a bias term.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+- `y`: is the target, which can be any table of responses whose element
+  scitype is `Continuous`; check the scitype with `scitype(y)`
+
+# Hyper-parameters
+
+- `bias=true`: Include the bias term if true, otherwise fit without bias term.
+
+# Operations
+
+- `predict(mach, Xnew)`: Return predictions of the target given new
+  features `Xnew` having the same Scitype as `X` above.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `coefficients`: The linear coefficients determined by the model.
+- `intercept`: The intercept determined by the model.
+
+# Examples
+
+```
+using MLJ
+using MLJBase: augment_X
```

Comment:
I think we should avoid `augment_X` as this is not public API. (Even I had to look it up). I'm guessing https://github.com/JuliaAI/MLJBase.jl/pull/780 will provide the alternative you want?

Reply:
yep! I will write those tests today :)